### ISSUE TITLE
[BUG] App does not register touches after tapping outside to dismiss popup

### ISSUE THREAD

**Issue by IraklisElef** [Mijick/Popups#149]
Hello, when I tap outside to dismiss the popup, the app stops registering touches and I have to reopen my app. However, if I put `.dismissAfter(3)`, this does not happen.

```
@main
struct ApolyomaiApp: App {
        
    //MARK: - PROPERTIES
    @UIApplicationDelegateAdaptor(AppDelegate.self) var delegate
    
    ...
    
    //MARK: - BODY
    var body: some Scene {
                
        WindowGroup {
            
            Group {
                
                if firstTimeOpenedApp {
                    
                    WelcomeUserView()
                    
                } else {
                    
                    ContentView()
                    
                } //: END IF
                
            } //: GROUP
            
        } //: WINDOW GROUP
        
    } //: VAR
    
} //: STRUCT 

class AppDelegate: NSObject, UIApplicationDelegate {
    
    ...
    
    func application(_ application: UIApplication, configurationForConnecting connectingSceneSession: UISceneSession, options: UIScene.ConnectionOptions) -> UISceneConfiguration {
        
           let sceneConfig = UISceneConfiguration(name: nil, sessionRole: connectingSceneSession.role)
           sceneConfig.delegateClass = CustomPopupSceneDelegate.self
           return sceneConfig
        
       } //: FUNC
    
} //: CLASS

class CustomPopupSceneDelegate: PopupSceneDelegate {
    
   override init() {
       
       super.init()
       
       configBuilder = { $0
           
           .vertical { $0
               
               .enableDragGesture(true)
               .tapOutsideToDismissPopup(true)
               .cornerRadius(32)
                
           } //: VERTICAL
           
           .centre { $0
               
               .tapOutsideToDismissPopup(true)
               .backgroundColor(.groupBox)
                
           } //: CENTRE
           
       } //: CONFIG
       
   } //: INIT
    
} //: CLASS
```

```
struct AchievementItemComponent: View {
    
    //MARK: - PROPERTIES
    ...
    
    //MARK: - BODY
    var body: some View {
        
        HStack(spacing: 15) {
            
            ...
            
        } //: HSTACK
        .onTapGesture {
                        
            AchievementDescriptionPopupComponent(achievement: achievement)
                .dismissAfter(3)
                .present()
            
        }
        
    } //: BODY
    
} //: STRUCT
```

```
struct AchievementDescriptionPopupComponent: CentrePopup {
    
    //MARK: - PROPERTIES
    @State public var achievement: ServiceAchievementModel
    
    //MARK: - FUNCTIONS
    func configurePopup(config: CentrePopupConfig) -> CentrePopupConfig {
        
        config
            .tapOutsideToDismissPopup(true)
            .popupHorizontalPadding(20)
            .cornerRadius(16)
        
    } //: FUNC
    
    //MARK: - BODY
    var body: some View {
        
        HStack(spacing: 0) {
            
            Text(achievement.description)

        } //: HSTACK
        .padding(.vertical, 10)
        .padding(.horizontal, 20)
        
    } //: BODY
    
} //: STRUCT
```

**Comment by FulcrumOne** [comment]
Hey @IraklisElef,

Thanks for the report. I'll look at this when I have some time, but currently I'm super busy updating the readme of this framework. I hope to finish it by Tuesday.

Have a nice day,
Tomasz

**Comment by FulcrumOne** [comment]
Hey @IraklisElef,

The problem seems to be resolved in the `patch-4.0.0` branch. Could you confirm this and let me know?

Thanks for your contribution and sorry that you had to wait for so long time,
Tomasz
